zmsdb/src/Zmsdb/Source/Dldb.php

```diff
@@ -40,6 +51,120 @@ public function startImport($verbose = true)
         \BO\Zmsdb\Connection\Select::writeCommit();
     }
 
+    /**
+     * Initiates the import process, includes conditional backup and data processing for dldb mapper data.
+     *
+     * @param bool $verbose Whether to log verbose output.
+     */
+    public function startDLDBImport($verbose = true)
+    {
+        if (!static::$importPath) {
+            throw new \Exception('No data path given');
+        }
+
+        if (static::$backupPath === '') {
+            throw new \Exception('Backup path not specified');
+        }
+
+        if ($verbose) {
+            self::$verbose = $verbose;
+            echo "Use source-path for dldb: " . static::$importPath . "\n";
+        }
+
+        if ($this->isBackupRequired()) {
+            $this->backupData();
```

Comment:
Ich glaube das ist falsche Stelle hier backup zu machen. Diese Methode macht import to DB anhand gespeicherte jsons. Richtige wäre zmsdldb/bin/dldbget (es ist php) wo man Datei erstellt da auch backup zu machen.

Reply:
Kannst du hier auch lassen, nicht so wichtig. :)

Reply:
moved from dldb.php to dldbget